swan-lake/learn/using-the-openapi-tools.md

```diff
@@ -53,35 +53,35 @@ If not, it will take the execution path as the output path.
 If you  want to generate a Service only, you can set the mode as `service` in the OpenAPI tool.
 
 ```bash
-ballerina openapi   -i <openapi-contract> --mode service
+bal openapi   -i <openapi-contract> --mode service
                                [(-o|--output) output file path]
 ```
 
 If you want to generate a Client only, you can set the mode as  `client` in the OpenAPI tool. 
 This client can be used in client applications to call the service defined in the OpenAPI file.
 
 ```bash
-ballerina openapi   -i <openapi-contract> --mode client
+bal openapi   -i <openapi-contract> --mode client
                                [(-o|--output) output file path]
 ```
 
 ### Ballerina to OpenAPI
 #### Service to OpenAPI Export
 ```bash
-ballerina openapi   -i <ballerina file> 
+bal openapi   -i <ballerina file> 
```

Comment:
This should be: 
bal openapi -i <ballerina-file-path>

The option names are inconsistent throughout this document.